client/src/components/ratingsReviews/RatingBreakdown.jsx

```diff
@@ -59,6 +64,8 @@ const RatingBreakdown = (props) => {
           <RatingProgress key={i} ratings={ratings} completed={item} handleRatingProgressFilter={handleRatingProgressFilter} />
         ))}
       </div>
+    ) : (
+      <div id='rr-rating-breakdown'>Loading...</div>
```

Comment:
Your feature uses tiles of a standard size, correct? If there are no reviews (yet?) maybe communicate that in something the size and shape of a review tile. Maybe show a skeleton of your rating breakdown so that the user will know what sort of data would be available from a review.

Reply:
Business document says to collapse all and show a button to add a review. I see your point, I'll keep the rating breakdown skeleton with 0 ratings, and on the right side i'll leave a button to add a review